src/vm/amd64/cgencpu.h

```diff
@@ -215,6 +195,16 @@ void UpdateRegDisplayFromCalleeSavedRegisters(REGDISPLAY * pRD, CalleeSavedRegis
 
 #define NUM_ARGUMENT_REGISTERS 6
 
+#define ENUM_CALLEE_SAVED_REGISTERS() \
```

Comment:
Could you please add a comment saying that the order of registers here must match the index of the register that is passed into RestoreReg macro?